src/main/webapp/mapScript.js

```diff
@@ -157,3 +157,7 @@ async function deletePlace(contentId) {
       }
     });
 }
+
+function hey() {
+  console.log("hey");
```

Comment:
You should probably delete this function

Reply:
Oh, sorry! I was checking why some of my functions were unreachable with that! I'll delete it right away.